xbmc/addons/PVRClient.h

```diff
@@ -905,6 +905,11 @@ namespace PVR
     bool LogError(const PVR_ERROR error, const char *strMethod) const;
 
     /*!
+     * @brief Write the given addon properties into given file item's properties.
```

Comment:
Would be good to add docs for params too.

Reply:
I think the @brief explains it good enough, adding documentation for the parameters is no real value add here. And it is a private method anyway.

Reply:
Ok I'll shut up now 😄

Reply:
Thanks anyway.